porcupine/plugins/anchors.py

```diff
@@ -0,0 +1,58 @@
+import tkinter
+import time
+from porcupine import tabs, get_tab_manager
+from porcupine.plugins import linenumbers
+
+# Dependent on code from linenumbers.py
+setup_after = ["linenumbers"]
+
+
+class AnchorManager:
+    def __init__(self, tab_textwidget: tkinter.Text, linenumber_instance: tkinter.Canvas) -> None:
+        self.tab_textwidget = tab_textwidget
+        self.linenumber_instance = linenumber_instance
```

Comment:
I would just name it `linenumbers`. Everything is an instance in Python, so the extra `_instance` adds no information.

If you want to avoid conflicting it with the global `linenumbers` variable introduced in `from porcupine.plugins import linenumbers`, you can change the import to `from porcupine.plugins.linenumbers import LineNumbers`. This creates a global variable named `LineNumbers`, which won't conflict with any lowercase variable names.